custom_components/tuya_local/devices/warmehaus_AFD02TJ_thermostat_c.yaml

```diff
@@ -0,0 +1,113 @@
+name: WarmeHaus Thermostatic Towel Rail Electric Heating Element
+products:
+  - id: fgzmbjxhgkuwgrav
+    manufacturer: WarmeHaus
+    model: AFD-02-TJ
+entities:
+  - entity: climate
+    translation_only_key: thermostat
+    dps:
+      - id: 1
+        type: boolean
+        name: hvac_mode
+        mapping:
+          - dps_val: false
+            value: 'off'
+          - dps_val: true
+            value: heat
+      - id: 2
+        type: string
+        name: preset_mode
+        mapping:
+          - dps_val: hot
+            value: Free mode
+          - dps_val: cold
+            value: Constant mode
+          - dps_val: eco
+            value: Boost
+          - dps_val: auto
+            value: Program
+      - id: 16
+        name: temperature
+        type: integer
+        unit: C
+        optional: true
+        range:
+          min: 300
+          max: 700
+        mapping:
+          - scale: 10
+            step: 10
+            constraint: preset_mode
+            conditions:
+              - dps_val: [cold, eco, auto]
+                invalid: true
+      - id: 24
+        type: integer
+        name: current_temperature
+      - id: 1
+        type: boolean
+        name: hvac_action
+        mapping:
+          - dps_val: true
+            value: heating
+          - dps_val: false
+            value: idle
+  - entity: sensor
+    translation_key: timer
+    class: duration
+    category: config
+    dps:
+      - id: 111
+        type: float
+        name: value
+        unit: h
+        range:
+          min: .5
+          max: 8
+        mapping:
+          - scale: 1
+            step: .5

```

Comment:
not yet working correctly. app shows time range between .5h and 8h. Need to figure out how to parse this.